pennylane/ops/op_math/controlled_decompositions.py

```diff
@@ -134,11 +135,94 @@ def _bisect_compute_b(u: np.ndarray):
     return _param_su2(c, d, b, 0)
 
 
-def ctrl_decomp_zyz(target_operation: Operator, control_wires: Wires):
+def _multi_controlled_zyz(
+    rot_angles,
+    global_phase,
+    target_wire: Wires,
+    control_wires: Wires,
+    work_wires: Optional[Wires] = None,
+) -> list[Operator]:
+    # The decomposition of special zyz with multiple control wires
+    # defined in Lemma 7.9 of https://arxiv.org/pdf/quant-ph/9503016
+
+    if not qml.math.allclose(0.0, global_phase, atol=1e-8, rtol=0):
+        raise ValueError(f"The global_phase should be zero, instead got: {global_phase}.")
```

Comment:
Why should the global phase be zero?

We are able to support the decomposition of a single-controlled global phase after https://github.com/PennyLaneAI/pennylane/pull/5735. What we're missing is decomposition of a multi-controlled global phase ([story here](https://app.shortcut.com/xanaduai/story/65125)).

Reply:
cause Lemma 7.9 is for special unitaries and I only need this support to unblock the QSVT projects in perf.
Lemma 7.5 is required to support non-zero global phase up to my understanding. I think you're right about [this story](https://app.shortcut.com/xanaduai/story/65125) but @albi3ro is the expert I know who can confirm the second part of your comment.